packages/lib/gpc/src/gpcCompile.ts

```diff
@@ -360,42 +362,64 @@ function compileProofListMembership<
   paramMaxListElements: number
 ): {
   listComparisonValueIndex: CircuitSignal[];
+  listContainsComparisonValue: CircuitSignal[];
```

Comment:
Is the bit-packing missing here?  I'd expect this to have been bit-packed into single CircuitSignal at this level, and I don't see code to do it anywhere else.